layers/state_tracker/descriptor_sets.h

```diff
@@ -983,7 +983,7 @@ class DescriptorSet : public StateObject {
         return DescriptorIterator<ConstBindingIterator>(*this, binding, index);
     }
 
-    virtual bool SkipBinding(const DescriptorBinding &binding, bool is_dynamic_accessed) const {
+    inline bool ValidateBindingOnGPU(const DescriptorBinding &binding, bool is_dynamic_accessed) const {
```

Comment:
```suggestion
   bool ValidateBindingOnGPU(const DescriptorBinding &binding, bool is_dynamic_accessed) const {
```
Function defined in a header are likely to be inlined anyway, and `inline` being just a hint, just let the compiler decide!

Reply:
Yeah, inline is mostly outdated now, compilers inline not only little but medium sized functions too, results in mostly large functions in the output, and function calls are less expensive as a result 🙃

Reply:
I guess another part of inline that deals with One Definition rule can more central reason to use it